src/content_mapblock.cpp

```diff
@@ -526,15 +539,16 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 			{
 				v3s16 dir = side_dirs[i];
 
+				NeighbourData& neighbour_data = neighbour_datas[NeightbourToIndex(dir)];
```

Comment:
NeightbourToIndex ... typo here (function should be called NeighbourToIndex() I believe)